src/tiled/tiled.pro

```diff
@@ -221,7 +221,9 @@ SOURCES += aboutdialog.cpp \
     zoomable.cpp \
     clickablelabel.cpp \
     imagecolorpickerwidget.cpp \
-    reversingproxymodel.cpp
+    reversingproxymodel.cpp \
+    objectiddialog.cpp \
+    objectidedit.cpp
```

Comment:
I know it's annoying that it doesn't do it automatically, but please keep these lists sorted. It helps avoid merge conflicts.